apps/store/extensions/app/store-apis/apis/taxonomy.jag

```diff
@@ -35,123 +35,122 @@ require('/modules/store.js').exec(function (ctx) {
     var username = user ? user.username : carbon.user.anonUser;
     var tenantId;
 
+
     var defaultPagination = function (options, key, limit) {
         return (options[key] ? options[key] : limit);
     };
 
-    switch (method) {
-        case 'GET':// GET endpoints
-            if (uriMatcher.match(TAXA)) {
-                acceptHeader = req.getHeader('Accept') || constants.ACCEPT_ALL;
-                jsonHeaderAccept = acceptHeader.indexOf('application/json');
-                try {
-                    if (jsonHeaderAccept >= 0 || acceptHeader == constants.ACCEPT_ALL) {
-                        var taxaPathList;
-                        var options = ref.getQueryOptions(req.getQueryString());
-                        var tenantDomain = options.tenant ? options.tenant : MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
-                        tenantId = carbon.server.tenantId({domain: tenantDomain});
-                        options.tenantId = tenantId;
-                        options.username = username;
-
-                        if (options['terms']) {
-                            store.server.sandbox(options, function () {
-                                taxaPathList = taxa.taxonomy.getNodesList(options['terms'],
-                                        defaultPagination(options, "startnode", 0),
-                                        defaultPagination(options, "endnode", 5));
-                            });
+        switch (method) {
+            case 'GET':// GET endpoints
+                if (uriMatcher.match(TAXA)) {
+                    acceptHeader = req.getHeader('Accept') || constants.ACCEPT_ALL;
+                    jsonHeaderAccept = acceptHeader.indexOf('application/json');
```

Comment:
Thank you Sameera ,
This has been fixed with this PR https://github.com/wso2/carbon-store/pull/834